src/nodes.js

```diff
@@ -61,14 +61,16 @@ var createNode = function(doc, nodeName, key, statics) {
   if (nodeName === '#text') {
     node = doc.createTextNode('');
   } else {
-    node = createElement(doc, nodeName, key, statics);
+    node = createElement(doc, nodeName, key);
   }
 
   initData(node, nodeName, key);
 
   if (statics) {
```

Comment:
This could actually be moved back into `createElement` with the `initData` call before it, but we'd need to add another `initData` for the TextNode path.